app/models/ability.rb

```diff
@@ -187,6 +188,7 @@ def signed_in_with_cfp_role(user)
     # ids of all the conferences for which the user has the 'cfp' role
     conf_ids_for_cfp = Conference.with_role(:cfp, user).pluck(:id)
 
+    can [:index, :show, :update], Resource, conference_id: conf_ids_for_cfp
```

Comment:
What I meant was that `update` include `edit` even if you don't add it explicitly, as CanCan has an alias for it. But I am perfectly fine with leaving it as it is now. :slightly_smiling_face: